crypto.h

```diff
@@ -220,6 +224,32 @@ typedef struct {
 /* just for consistency */
 #define dtls_kb_digest_size(Param, Role) DTLS_MAC_LENGTH
 
+#ifdef DTLS_ATECC608A
+/**
+ * @brief Slot id used to perform ECDHE operation. 
+ *        Due to the last 'E', the operation is performed with 
+ *        ephemeral keys that must be generated and stored in the
+ *        the current slot.
+ *        I advise to use slot configured as follows:
+ *          - SlotConfig[slot_id] = 0x2087
+ *          - KeyConfig[slot_id] = 0x0013
+ *        When using test_ecc608_configdata of cryptoauthlib tests, 
+ *        it corresponds to slot 2.
+ * @warning Slot ID must be different from ecc_slot_id.
+ */
+extern uint8_t ecdhe_slot_id;
+
+/**
+ * @brief Slot id used to perform ECDSA operation.
+ *        This slot must contains the private key used to sign the
+ *        message. The associated public key is used to verify the signature.
+ *        When using test_ecc608_configdata of cryptoauthlib tests, 
+ *        you can use slot 7.
+ * @warning Slot ID must be different from ecdhe_slot_id.
+ */
+extern uint8_t ecc_slot_id;
+#endif
```

Comment:
As short term work around, maybe.
But I guess, if a peer executes 2 handshakes simultaneously, that will fail.

Reply:
CryptoAuthLib offers a mutex implementation. However, I suspect that if the handshake takes too long, a connection might fail due to timeout?